### PR TITLE
params_api_test.c: Fix mistake in backported test fix

### PR DESCRIPTION
**Pull request by t8m**
Fixup for e8f1d76b50204d87a0ef7f6879eb1dd507a54368.

Fixes on-push CI failure on 3.1 branch. Marking urgent.

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

File: test/params_api_test.c [hunk 1]
```diff
@@ -431,7 +431,7 @@ static int test_param_bignum(int n)
 
     if (!TEST_true(OSSL_PARAM_set_BN(&param, b)))
         goto err;
-    le_copy(buf, bnbuf, len);
+    le_copy(buf, bnbuf, sizeof(bnbuf));
```
Comment (slontis) [inline test/params_api_test.c:434]: Memseting buf to zero also would have worked right?

Comment (t8m) [reply test/params_api_test.c:434]: IMO no

**Comment by t8m** [comment]
ping for second review

**Comment by t8m** [comment]
ping for second review, this is urgent as the on-push CI big endian cross-compile targets are broken

**Review by beldmit — APPROVED** [review]
LGTM, agree urgent

**Pull request closed without merging**